packages/paste-core/components/blockquote/src/Blockquote.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import type { BoxProps } from "@twilio-paste/box";
+import { BlockquoteIcon } from "@twilio-paste/icons/esm/BlockquoteIcon";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import React from "react";
+
+import { BlockquoteContext } from "./BlockquoteContext";
+
+export interface BlockquoteProps extends HTMLPasteProps<"div"> {
+  children?: React.ReactNode;
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   * @default 'BLOCKQUOTE'
+   * @type {BoxProps['element']}
+   * @memberof BlockquoteProps
+   */
+  element?: BoxProps["element"];
+  /**
+   * The URL to the source of the quote
+   * @type {string}
+   * @memberof BlockquoteProps
+   */
+  url?: string;
+}
+
+export const Blockquote = React.forwardRef<HTMLDivElement, BlockquoteProps>(
+  ({ children, element = "BLOCKQUOTE", url, ...props }, ref) => {
+    return (
+      <Box
+        {...safelySpreadBoxProps(props)}
+        ref={ref}
+        display="flex"
+        columnGap="space50"
+        alignItems="flex-start"
+        lineHeight="lineHeight30"
+        fontSize="fontSize30"
+        element={element}
+      >
+        <BlockquoteIcon element={`${element}_ICON`} decorative={true} color="colorTextIcon" />
+        <BlockquoteContext.Provider value={{ url }}>
+          <Box element={`INNER_${element}`}>{children}</Box>
```

Comment:
Did we decide whether this was needed? Does it do weird stuff because the parent is a flex container?

Reply:
yeah exactly, it's needed for flex reasons, so I added a customization target and called it a day